### PR TITLE
feat: transparent video

### PR DESCRIPTION
**Pull request by hkonsti**

### CODE REVIEW

**Comment by justusmattern27** [comment]
nice work!

File: packages/renderer/server/renderVideo.ts [hunk 1]
```diff
@@ -215,19 +215,13 @@ async function cleanup(
   );
 
   const folderCleanupPromises = cleanupFolders.map(folder =>
-    fs.promises
-      .rm(folder, {recursive: true, force: true})
-      .catch(err =>
-        console.error(`Error during cleanup, couldn't remove ${folder}:`, err),
-      ),
+    // swallow error if file doesn't exist
```
Comment (justusmattern27) [inline packages/renderer/server/renderVideo.ts:218]: comment intentional?

File: packages/renderer/server/renderVideo.ts [hunk 2]
```diff
@@ -215,19 +215,13 @@ async function cleanup(
...
+    fs.promises.rm(folder, {recursive: true, force: true}).catch(),
   );
 
   const fileCleanupPromises = cleanupFiles.map(file =>
-    fs.promises
-      .unlink(file)
-      .catch(err =>
-        console.error(`Error during cleanup, couldn't remove ${file}:`, err),
-      ),
+    // swallow error if file doesn't exist
```
Comment (justusmattern27) [inline packages/renderer/server/renderVideo.ts:223]: comment intentional?

File: packages/2d/src/lib/components/Video.ts [hunk 3]
```diff
@@ -278,6 +278,7 @@ export class Video extends Media {
 
     if (playbackState === PlaybackState.Rendering) {
       return this.serverSeekedVideo();
+      // return this.seekedVideo();
```
Comment (justusmattern27) [inline packages/2d/src/lib/components/Video.ts:281]: I think this comment should be removed

**Review by justusmattern27 — APPROVED** [review]

File: packages/2d/src/lib/components/Video.ts (hunk 3 above)
Comment (hkonsti) [reply packages/2d/src/lib/components/Video.ts:281]: oh oops you're right hahah

**Pull request merged**